src/ape/cli/options.py

```diff
@@ -116,7 +116,7 @@ def network_option(default: str = networks.default_ecosystem.name):
           use as the default. Defaults to how ``ape`` normally
           selects a default network.
     """
-
+    default = default or networks.default_ecosystem.name
```

Comment:
Step 1 of New Years laziness:  Calculate the default runtime more lazily for this CLI option (this was required to get the tests to work)